app-misc/anki/anki-25.02.4.ebuild

```diff
@@ -286,6 +274,18 @@ python_install_all() {
 }
 
 src_install() {
+	local DOC_CONTENTS="Users with add-ons that still rely on Anki's Qt5 GUI
+	can temporarily set the environment variable ENABLE_QT5_COMPAT to 1 to have
+	Anki install the previous compatibility code. This option has additional
+	runtime dependencies. Please take a look at this package's optional runtime
+	features for a complete listing.
+	\n\nENABLE_QT5_COMPAT may be removed in the future, so this is not a
+	long-term solution.
+	\n\nAnki's user manual is located online at https://docs.ankiweb.net/
+	\nAnki's add-on developer manual is located online at
+	https://addon-docs.ankiweb.net/"
+
+	readme.gentoo_create_doc
```

Comment:
You could consider switching to `greadme.eclass`, which allows to create the readme via Here Documents, resulting in improve readability.

Reply:
Thanks for the review! I switched to `greadme.eclass` as it also allowed me to easily customize the installed README based on the USE flag 'gui'.

Reply:
@Flowdalic In future, please consider making recommendations like that w/ a caveat or note that it should be done later. A larger refactor in something which should be stabled quickly for a security bug isn't ideal.

Reply:
> @Flowdalic In future, please consider making recommendations like that w/ a caveat or note that it should be done later. A larger refactor in something which should be stabled quickly for a security bug isn't ideal.


I would have done that, had I known that this PR was security related. The PR title says "create README.gentoo before attempting to install it; drop old". It's only noticable from the linked bugs.



> Thanks for the review! I switched to `greadme.eclass` as it also allowed me to easily customize the installed README based on the USE flag 'gui'.

Great to see that you find greadme.eclass useful. 😀

Reply:
>> In future, please consider making recommendations like that w/ a caveat or note that it should be done later. A larger refactor in something which should be stabled quickly for a security bug isn't ideal.

> I would have done that, had I known that this PR was security related.

Apologies for the mediocre PR title on my part. To be honest, I wasn't too sure what the `B2 [stable?]` remark on https://bugs.gentoo.org/954770 meant in detail and what it implied for other open bugs for the package. I should have clarified that first before moving forward with this PR.

Reply:
> I would have done that, had I known that this PR was security related

The bot adds a "security" label to the PR.

Reply:
> The bot adds a "security" label to the PR.

Thanks. And I am sorry I did not notice that before.